test/integration/SubscriberResends.test.ts

```diff
@@ -18,20 +20,21 @@ const MAX_MESSAGES = 5
 describeRepeats('resends', () => {
     let expectErrors = 0 // check no errors by default
     let onError = jest.fn()
-    let client
-    let stream
-    let published
-    let publishedRequests
-    let publishTestMessages
-    let waitForStorage
-    let subscriber
+    let client: StreamrClient
+    let stream: Stream
+    let published: any[]
+    let publishedRequests: any[]
+    let publishTestMessages: ((n?: number, opts?: any) => Promise<[message: any, request: any][]>) & { raw: (...args: any[]) => any }
```

Comment:
Yeah. If we get this kind of complex return types (a function which returns a Promise of array + has additional property `raw`), it would make sense to convert that functionality to a class or something.

Reply:
sure, but not sure it really changes the typing much though